cdc/entry/schema/snapshot.go

```diff
@@ -194,9 +200,43 @@ func NewEmptySnapshot(forceReplicate bool) *Snapshot {
 		forceReplicate:   forceReplicate,
 		currentTs:        0,
 	}
+
 	return &Snapshot{inner: inner, rwlock: new(sync.RWMutex)}
 }
 
+// these constants imitate TiDB's session.InitDDLJobTables in an empty Snapshot.
+const (
+	mysqlDBID = int64(1)
+	dummyTS   = uint64(1)
+)
+
+// InitConcurrentDDLTables imitates the creating table logic for concurrent DDL.
+// Since v6.2.0, tables of concurrent DDL will be directly written as meta KV in
+// TiKV, without being written to history DDL jobs. So the Snapshot which is not
+// build from meta needs this method to handle history DDL.
+func (s *Snapshot) InitConcurrentDDLTables() {
```

Comment:
Should we add a test for this func? Or any tests can cover this bug?

Reply:
old unit tests will fail